### ISSUE TITLE
Keras training accuracy is always equal to 1

### ISSUE THREAD

**Issue by ChrisWPC** [keras-team/keras#14582]
![acc = 1](https://user-images.githubusercontent.com/71392051/113894489-0afa5a80-9796-11eb-8d65-fe5e47c36b55.png)

Hey y'all, I was training my model to predict if the picture given was a cat or a dog. However I am getting some issues with the accuracy, as you can see it stays constant on the number one. I do not see the issue here, could someone help me get the accuracy to show the correct value?

**Comment by bguetarni** [comment]
I suspect your model to **overfit** your data. This is not an error, your model is literally perfect on your training data because it has learned the label for each one of your images. However overfitting is not a good thing because your model could poorly predict on new images. To check this, extract a small part of your data (let's say 20% - called validation set), and after training evaluate your model on this extracted part. If the accuracy is low, it means that your model is overfitting. Otherwise it's actually a good model.

**Comment by ChrisWPC** [comment]
Funny thing happened actually. My data got split into two files meaning the directory I had placed for accessing the data was incomplete. So after noticing this and fixing the error, the model gave me the accuracy I was looking for, and it no longer overfit. Thank you for taking your time to respond!

**Comment by sushreebarsa** [comment]
@ChrisWPC Could you please let us know if this issue still persists and if it is resolved then please move this issue to closed status? Thank you!

**Comment by sushreebarsa** [comment]
Closing this issue due to lack of recent activity.Please feel free to reopen if you still have concern.Thanks!
